packages/create-astro/src/logger.ts

```diff
@@ -0,0 +1,141 @@
+import { bold, blue, dim, red, yellow } from 'kleur/colors';
+import { Writable } from 'stream';
+import { format as utilFormat } from 'util';
+
+type ConsoleStream = Writable & {
+	fd: 1 | 2;
+};
+
+function getLoggerLocale(): string {
+	const defaultLocale = 'en-US';
+	if (process.env.LANG) {
+		const extractedLocale = process.env.LANG.split('.')[0].replace(/_/g, '-');
+		// Check if language code is atleast two characters long (ie. en, es).
+		// NOTE: if "c" locale is encountered, the default locale will be returned.
+		if (extractedLocale.length < 2) return defaultLocale;
+		else return extractedLocale;
+	} else return defaultLocale;
+}
+
+const dt = new Intl.DateTimeFormat(getLoggerLocale(), {
+	hour: '2-digit',
+	minute: '2-digit',
+});
+
+export const defaultLogDestination = new Writable({
+	objectMode: true,
+	write(event: LogMessage, _, callback) {
+		let dest: ConsoleStream = process.stderr;
+		if (levels[event.level] < levels['error']) {
+			dest = process.stdout;
+		}
+
+		dest.write(dim(dt.format(new Date()) + ' '));
+
+		let type = event.type;
+		if (type) {
+			if (event.level === 'info') {
+				type = bold(blue(type));
+			} else if (event.level === 'warn') {
+				type = bold(yellow(type));
+			} else if (event.level === 'error') {
+				type = bold(red(type));
+			}
```

Comment:
```suggestion
   switch (event.level) {
      case 'info': type = bold(blue(type));
      case 'warn': type = bold(yellow(type));
      case 'error': type = bold(red(type));
    }
```

note: untested

Reply:
I added that and added some `break`'s to it, thanks!

Reply:
thanks!